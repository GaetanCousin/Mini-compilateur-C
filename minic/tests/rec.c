extern int printf();

void f(int n) {

        long a;
        long b;
        long c;
        int i;
        for (i = 0; i < 10; i++) {
                int j;
                long t;
                t = 4;
                printf("2t: %d\n", 2*t);
        }
}


int main () {

        f(30);
        return 0;
}
