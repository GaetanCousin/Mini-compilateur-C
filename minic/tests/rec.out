2t: 8
2t: 8
2t: 8
2t: 8
2t: 8
2t: 8
2t: 8
2t: 8
2t: 8
2t: 8
